tools/accuracy_checker/openvino/tools/accuracy_checker/metrics/background_matting.py

```diff
@@ -159,7 +161,7 @@ def update(self, annotation, prediction):
         if pred.shape[-1] == 1 and pred.shape[-1] != gt.shape[-1]:
             gt = cv2.cvtColor(gt, cv2.COLOR_RGB2GRAY)
         if self.use_mask:
-            mask = self.prepare_pha(annotation.value) > 0
+            mask = self.prepare_pha(annotation) > 0.01
```

Comment:
can we have some parameter for changing tolerance?

Reply:
added new parameter